src/Administration/Resources/app/administration/src/module/sw-mail-template/page/sw-mail-template-index/index.js

```diff
@@ -1,25 +1,30 @@
 import template from './sw-mail-template-index.html.twig';
 
-const { Component, Mixin } = Shopware;
+const { Component } = Shopware;
 
 Component.register('sw-mail-template-index', {
     template,
 
-    mixins: [
-        Mixin.getByName('listing')
-    ],
-
     metaInfo() {
         return {
             title: this.$createTitle()
         };
     },
 
+    data() {
+        return {
+           searchTerm: undefined
```

Comment:
I would set an empty string here. This is also done by most search implementations inside the admin.

```suggestion
           searchTerm:''
```